src/components/UI/IIIFImageEmbedModal.js

```diff
@@ -0,0 +1,232 @@
+                  className={`button big ${width === 3000 ? "active" : ""}`}
+                  onClick={() => handleWidthChange(3000)}
+                >
+                  3000 pixels (100%)
+                </button>
+              </p>
+              <p>
+                <button
+                  className={`button medium ${width === 1800 ? "active" : ""}`}
+                  onClick={() => handleWidthChange(1800)}
+                >
+                  1800 pixels (500%)
+                </button>
+              </p>
+              <p>
+                {" "}
+                <button
+                  className={`button ${width === 900 ? "active" : ""}`}
+                  onClick={() => handleWidthChange(900)}
+                >
+                  900 pixels (250%)
+                </button>
+              </p>
+              <p>
+                <button
+                  className={`button small ${width === 450 ? "active" : ""}`}
+                  onClick={() => handleWidthChange(450)}
+                >
+                  450 pixels (12%)
+                </button>
+              </p>
+            </div>
+
+            <fieldset>
+              <legend css={legend}>
+                <strong>Image color</strong>
+              </legend>
+              <input
+                id="bitonal"
+                name="color"
+                type="radio"
+                value="bitonal"
+                onChange={handleColorChange}
+                checked={color === "bitonal"}
+              />
+              <label className="inline" htmlFor="bitonal">
+                Bitonal
+              </label>
+              <input
+                id="gray"
+                name="color"
+                type="radio"
+                value="gray"
+                onChange={handleColorChange}
+                checked={color === "gray"}
+              />
+              <label className="inline" htmlFor="gray">
+                Gray
+              </label>
+              <input
+                id="color"
+                name="color"
+                type="radio"
+                value="color"
+                onChange={handleColorChange}
+                checked={color === "color"}
+              />
+              <label className="inline" htmlFor="color">
+                Color
+              </label>
+            </fieldset>
+          </div>
+          <div css={column}>
+            <h3>HTML Tag</h3>
+            <textarea readOnly value={embedCode} />
+            {copied && <p css={alert}>Copied</p>}
+          </div>
+        </div>
+        <div css={footer}>
+          {copied ? (
+            <button className="button" onClick={() => closeModal()}>
+              Close
+            </button>
+          ) : (
+            <div>
+              <button className="button-link" onClick={() => closeModal()}>
+                Cancel
+              </button>
+              <CopyToClipboard text={embedCode} onCopy={() => setCopied(true)}>
```

Comment:
I like that it saves the state for the type of image I want to download but it is also seems to save `Copied` option on all the images if I clicked `Copy to Clipboard` on one of the images. Can we create an option to cancel the copy? And also clear the copy when a new instance of `Embed` modal is opened?

Reply:
Good catch.  Yes, will update to clear out the saved clipboard info.